CHANGELOG.md

```diff
@@ -7,6 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Coming soon
 * TBD
 
+## Version 7.3.0 (2021-9-30)
+
+### New features and improvements
+
+* Support for [Java 16](https://github.com/newrelic/newrelic-java-agent/pull/366)
+
+* Auto-instrumentation support for [java.net.http.HttpClient](https://github.com/newrelic/newrelic-java-agent/pull/251) 
+
+* Migrate the Agent’s caching library from Guava to [Caffeine](https://github.com/newrelic/newrelic-java-agent/pull/295)
+  * [Caffeine provides an in-memory cache](https://github.com/ben-manes/caffeine#cache) using a Google Guava inspired API. The improvements draw on the author’s experience designing Guava's cache and ConcurrentLinkedHashMap.
+  * This change will provide improvements in cases where we saw thread contention and deadlocks attributable to the Guava library
```

Comment:
```suggestion
  * This change will provide improvements in cases where we saw thread contention and deadlocks attributable to the Guava library.
```

("will" or "is expected to" :thinking:)

Reply:
Good call. "Is expected" would be more appropriate here.